_assets/stylesheets/pages/_tv.scss

```diff
@@ -0,0 +1,42 @@
+.tv-btns {
+  width: 182px;
+  height: 54px;
+}
```

Comment:
I have these same buttons on /watch. Just use what I already have for this jumbotron.
https://github.com/crdschurch/crds-net/blob/master/_assets/stylesheets/pages/_watch.scss